bundles/org.eclipse.equinox.preferences/META-INF/MANIFEST.MF

```diff
@@ -11,7 +11,7 @@ Require-Bundle: org.eclipse.equinox.common;bundle-version="[3.18.0,4.0.0)",
  org.osgi.service.prefs;bundle-version="[1.1.0,1.2.0)";visibility:=reexport
 Export-Package: org.eclipse.core.internal.preferences;x-friends:="org.eclipse.core.resources,org.eclipse.core.runtime,org.eclipse.equinox.p2.engine",
  org.eclipse.core.internal.preferences.exchange;x-friends:="org.eclipse.core.runtime",
- org.eclipse.core.runtime.preferences;version="3.5.0";uses:="org.eclipse.core.runtime,org.osgi.service.prefs,org.eclipse.core.internal.preferences"
+ org.eclipse.core.runtime.preferences;version="3.5.0";uses:="org.eclipse.core.internal.preferences,org.eclipse.core.runtime,org.osgi.service.prefs"
```

Comment:
Need to check if this is maybe wrong that the internal package is listed at all (what would mean API leakage)

Reply:
@tjwatson I think this is because

`org.eclipse.core.runtime.preferences.BundleDefaultsScope` extends `org.eclipse.core.internal.preferences.AbstractScope` (what is also used by `org.eclipse.core.resources.ProjectScope`).

This could be solved by copy `AbstractScope` into the public API and deprecate the old one, the class it self really does not do much other than give implicitly access to the (internal) `PreferencesService`.